atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraTimestampBoundStore.java

```diff
@@ -194,4 +223,58 @@ private Column makeColumn(long ts) {
     private static ByteBuffer getRowName() {
         return ByteBuffer.wrap(PtBytes.toBytes(ROW_AND_COLUMN_NAME));
     }
+
+    private String replaceBracesWithStringFormatSpecifier(String msg) {
+        return msg.replaceAll("\\{\\}", "%s");
+    }
+
+    private String getCurrentTimestampValues(CASResult result) {
+        return result.current_values.stream()
+                .map(Column::getValue)
+                .map(PtBytes::toString)
+                .collect(Collectors.joining(", "));
+    }
+
+    @VisibleForTesting
+    long getId() {
+        return id;
+    }
+
+    private static final class IdAndTimestamp {
```

Comment:
It's already in your branch: https://github.com/palantir/atlasdb/blob/fix/mrtse/atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/persistentlock/LockEntry.java

The refactor is somewhat beyond the scope of this PR - we should chat to @jboreiko about it.